test/spec/modules/fledge_spec.js

```diff
@@ -35,3 +42,166 @@ describe('fledgeForGpt module', function() {
     });
```

Comment:
prob does not matter but this is still calling `bidRequest` instead of `adUnitCode` for `addComponentAuctionHook`

Line 40 in `fledge_spec`